h2o-algos/src/main/java/hex/ensemble/StackedEnsemble.java

```diff
@@ -303,10 +304,10 @@ private void computeMetaLearner(Frame levelOneTrainingFrame, Frame levelOneValid
           if (_parms._keep_levelone_frame) {
             _model._output._levelone_frame_id = levelOneTrainingFrame; //Keep Level One Training Frame in Stacked Ensemble model object
           } else{
-            DKV.remove(levelOneTrainingFrame._key); //Remove Level One Training Frame from DKV
+            //DKV.remove(levelOneTrainingFrame._key); //Remove Level One Training Frame from DKV
           }
           if (null != levelOneValidationFrame) {
-            DKV.remove(levelOneValidationFrame._key); //Remove Level One Validation Frame from DKV
+            //DKV.remove(levelOneValidationFrame._key); //Remove Level One Validation Frame from DKV
```

Comment:
So this will keep the level one frames around even if a user doesn't want the frames? Why comment this out?

Reply:
We added this while debugging and I think we determined that it was necessary.  I will look into whether we can remove it (can follow up in fix release).